frontend/src/pages/projects/screens/detail/notebooks/NotebookTableRow.tsx

```diff
@@ -31,11 +31,11 @@ const NotebookTableRow: React.FC<NotebookTableRowProps> = ({
     <>
       <Tr>
         <Td expand={{ rowIndex: 0, isExpanded, onToggle: () => setExpanded(!isExpanded) }} />
-        <Td>
+        <Td dataLabel="Name">
```

Comment:
hmmm -- I'd imagine it's always `data-` even in JSX 🤔

Reply:
This is a built-in PatternFly prop, which will finally set `data-label` in the HTML tag, I have already verified this.